### PR TITLE
Fixing the visual issues for the toolbar and sidebar

### PR DESCRIPTION
**Pull request by Manarnz**
Previously, the config sidebar would not scroll down. Instead, it would keep going downwards everytime a new request is made. It is not exactly issue #477, as the issue changed once the "Started Configurations Revamp in new branch" PR was mereged.

### CODE REVIEW

**Review by leshghi — CHANGES_REQUESTED** [review]
I'm not sure what Issue you're working on - is this supposed to be issue #477 that you were assigned? The problem is actually that the logo hides the configurations even after you scroll down. Scrolling has already been enabled for the configs. It would be great to bring the div that holds the config sidebar up above the logo somehow so there is no overlap, while also extending the length of the overall side panel so that when you scroll down on the webpage as a whole there is no gap. See picture in issue for reference.

**Comment by Manarnz** [comment]
> I'm not sure what Issue you're working on - is this supposed to be issue #477 that you were assigned? The problem is actually that the logo hides the configurations even after you scroll down. Scrolling has already been enabled for the sidebar. It would be great to bring the div that holds the config sidebar up above the logo somehow so there is no overlap.

After Angela and I merged our PR, the configInspector would not scroll down. Instead it just kept incrementing downwards as demonstrated in the photo. 
image
I fixed this issue in this PR. This is why I stated that it is not exactly issue #477. I will now work on the logo problem, thank you for the clarification!

**Comment by leshghi** [comment]
> > I'm not sure what Issue you're working on - is this supposed to be issue #477 that you were assigned? The problem is actually that the logo hides the configurations even after you scroll down. Scrolling has already been enabled for the sidebar. It would be great to bring the div that holds the config sidebar up above the logo somehow so there is no overlap.
> 
> After Angela and I merged our PR, the configInspector would not scroll down. Instead it just kept incrementing downwards as demonstrated in the photo.
> image
> I fixed this issue in this PR. This is why I stated that it is not exactly issue #477. I will now work on the logo problem, thank you for the clarification!

Ah got it - In that case can you make an actual issue for that as well and tag it in this PR? If possible it would be awesome if you could work on #477 on this PR as well since the two are related.

**Comment by leshghi** [comment]
> > > I'm not sure what Issue you're working on - is this supposed to be issue #477 that you were assigned? The problem is actually that the logo hides the configurations even after you scroll down. Scrolling has already been enabled for the sidebar. It would be great to bring the div that holds the config sidebar up above the logo somehow so there is no overlap.
> > 
> > 
> > After Angela and I merged our PR, the configInspector would not scroll down. Instead it just kept incrementing downwards as demonstrated in the photo.
> > image
> > I fixed this issue in this PR. This is why I stated that it is not exactly issue #477. I will now work on the logo problem, thank you for the clarification!
> 
> Ah got it - In that case can you make an actual issue for that as well and tag it in this PR? If possible it would be awesome if you could work on #477 on this PR as well since the two are related.

Also - there is no issue 744, so could you please remove the references to that in this PR? Thanks!

**Review by leshghi — CHANGES_REQUESTED** [review]
Have you tested this? The configs still overlap with the logo. This might involve a larger css fix/re-arrangement

**Comment by YesugenB** [comment]
When you go to Analysis view, the Attribution section disappears, and is not there when you go back to Modeling view.

**Comment by Manarnz** [comment]
> When you go to Analysis view, the Attribution section disappears, and is not there when you go back to Modeling view.

It is supposed to disappear in analysis mode. As for the second part, for some reason it is working for me. Could you please check now?

**Comment by YesugenB** [comment]
Yup it's fixed now! Thanks

File: leaf-ui/css/style.css [hunk 1]
```diff
@@ -172,7 +169,7 @@ body.waiting *
 .inspector textarea {
    color: black;
    text-shadow: none;
-   box-shadow: inset 0 0 0 1px rgba(0,0,0,0.1),0px 0 0 1px rgba(255,255,255,0.1);
+   /* box-shadow: inset 0 0 0 1px rgba(0,0,0,0.1),0px 0 0 1px rgba(255,255,255,0.1); */
```
Comment (YesugenB) [inline leaf-ui/css/style.css]: Can this be removed now?

File: leaf-ui/index.html [hunk 2]
```diff
@@ -80,39 +77,45 @@
             </div>
         </div>
 
+        <button id="analysis-btn" class="green-btn btn model-only" style="border-radius:none;">Analysis</button>
+        <button id="modeling-btn" class="red-btn btn analysis-only" style='display: none';>Modeling</button>
```
Comment (YesugenB) [inline leaf-ui/index.html]: Can we add `""` around `style` instead of `''` so it's consistent

Comment (YesugenB) [reply leaf-ui/index.html]: And `none';` should be `none;'`?

File: leaf-ui/index.html [hunk 3]
```diff
@@ -80,39 +77,45 @@
...
 
-        <button id="btn-view-assignment" class="blue-btn btn model-only"
+        <button id="btn-view-assignment" class="btn model-only"
             style="position:relative; border-radius:none;">View Assignments List</button>
         <!--This is the modal box of assignments-->
-        <button id="btn-view-intermediate" class="blue-btn btn model-only"
-            style="position:relative; border-style:none none none solid; border-color: rgb(74, 105, 245); border-width: 1px">Intermediate
+        <button id="btn-view-intermediate" class="btn model-only">Intermediate
             Values</button>
         <!--This is the modal box of Intemdiate Values-->
```
Comment (YesugenB) [inline leaf-ui/index.html]: `Intermediate`

File: leaf-ui/index.html [hunk 4]
```diff
@@ -80,39 +77,45 @@
...
 
-        <button id="analysis-btn" class="green-btn btn model-only" style="border-radius:none;">Analysis</button>
-
-        <button id="simulate-single-path-btn" ; class="dark-green-btn btn analysis-only" ;
+        <button id="simulate-single-path-btn" ; class="btn analysis-only" ;
```
Comment (YesugenB) [inline leaf-ui/index.html]: I'm not sure that we need the` ;` between id and class in this and line 111.

File: leaf-ui/index.html [hunk 5]
```diff
@@ -80,39 +77,45 @@
...
             style="border-radius: none; display: none" ;>Simulate Path</button>
-        <button id="next-state-btn" ; class="light-green-btn btn analysis-only"
+        <button id="next-state-btn" ; class="btn analysis-only"
             style="border-radius: none; display: none" ;>Explore States</button>
-        <button id="modeling-btn" ; class="red-btn btn analysis-only"
-            style='display: none';>Modeling</button>
+       <!-- <div id="dropdown-model" class="dropdown analysis-only" style='display: none'>
```
Comment (YesugenB) [inline leaf-ui/index.html]: Change quotation marks around `style`

**Review by YesugenB — APPROVED** [review]
Looks good

**Review by amgrubb — APPROVED** [review]
Nice work, it looks great! Please address @YesugenB's comments.

**Review by amgrubb — CHANGES_REQUESTED** [review]
Sorry, clicked the wrong button. Please address @YesugenB's comment.

File: leaf-ui/index.html (hunk 5 above)
Comment (Manarnz) [reply leaf-ui/index.html]: This is commented out.

**Review by amgrubb — APPROVED** [review]
Excellent, thanks for making these changes.

**Pull request merged**